addon-test-support/@percy/ember/index.js

```diff
@@ -77,33 +77,49 @@ export default async function percySnapshot(name, options = {}) {
   if (!agentJS) return;
 
   let scopedSelector = options.scope || '#ember-testing';
-  let script = document.createElement('script');
-  script.innerText = agentJS;
-  document.body.appendChild(script);
+  let $script = document.querySelector('.percy-agent-js');
+
+  if (!$script) {
+    $script = document.createElement('script');
+    $script.classList.add('percy-agent-js');
+    $script.innerText = agentJS;
```

Comment:
Should we only fetch the DOM library here rather than on every snapshot? (`fetchDOMLib`)

Reply:
`fetchDOMLib` shortcuts if the JS has already been fetched (`if (agentJS) return agentJS;`)

Reply:
Gotcha